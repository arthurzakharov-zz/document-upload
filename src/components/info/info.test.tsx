import { render } from "@testing-library/react";
import Info from "./info.component";

describe("Info", () => {
  test("Snapshot", () => {
    const { container } = render(<Info />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
