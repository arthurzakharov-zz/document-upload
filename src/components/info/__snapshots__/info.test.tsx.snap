// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Snapshot 1`] = `
<div
  class="info"
>
  <h1
    class="info__title"
  >
    Dokumentenübermittlung
  </h1>
  <div
    class="info__info"
  >
    <div
      class="info__label"
      data-testid="info-number"
    >
      <b>
        Aktenzeichen: 
      </b>
      SBE22.0121-24046542
    </div>
    <div
      class="info__label"
      data-testid="info-name"
    >
      <b>
        Name: 
      </b>
      Luca Zamai
    </div>
  </div>
  <p
    class="info__description"
  >
    Klicken Sie auf die orangenen Buttons, um die jeweiligen Dateien zu übermitteln.
  </p>
</div>
`;
